.travis.yml

```diff
@@ -63,7 +64,8 @@ script:
   - yarn run lint
   - yarn run lint:css
   - if [[ $TRAVIS_BRANCH != $TRAVIS_TAG ]]; then yarn build ; fi
-  - if [[ $TRAVIS_BRANCH == $TRAVIS_TAG ]]; then ionic cordova platform add android;
+  - if [[ $TRAVIS_BRANCH == $TRAVIS_TAG ]]; then cordova prepare;
+    yarn add rxjs;
```

Comment:
why do we need to add rxjs explicitly?

Reply:
For some reason, when plugins are installed `rxjs` gets uninstalled. I will check if it is a module incompatibility issue.